src/test/java/core/basesyntax/dao/impl/FruitDaoImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.dao.impl;
+
+import static core.basesyntax.db.Storage.storage;
+
+import core.basesyntax.dao.FruitDao;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    private final FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
it's better to replace initialization to BeforeEach or BeforeAll method
check other places